tensorrt_llm/_torch/modules/fla/utils.py

```
@@ -0,0 +1,326 @@
+
+    warnings.warn(
+        ("Triton is not supported on current platform, roll back to CPU."),
+        stacklevel=1)
+
+
+@lru_cache(maxsize=None)
+def get_multiprocessor_count(tensor_idx: int = 0) -> int:
+    try:
+        return triton.runtime.driver.active.utils.get_device_properties(
+            tensor_idx)["multiprocessor_count"]
+    except BaseException:
```

Ruff (0.13.1)

Do not catch blind exception: `BaseException`

(BLE001)